DistroLauncher/PatchConfig.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Ubuntu
```

Comment:
Everything in this namespace could be members of the `PatchConfig` object, this way their purpose is more evident.

After all, the Patcher is only used by the `PatchConfig`, and the typdefs are only relevant to it.